app/scripts/modules/core/src/pipeline/executions/execution/ExecutionInformationPopoverState.ts

```diff
@@ -0,0 +1,56 @@
+import { API } from 'core/api/ApiService';
+import { ExecutionsTransformer } from '../../service/ExecutionsTransformer';
+import { IExecution, IPipeline } from 'core/domain';
+
+export class ExecutionInformationPopoverState {
```

Comment:
@maggieneterval since a pipeline execution stage can call another pipeline that may exist in another application that data is not available since deck is loading an `application` and storing data on that object each time you open the pipelines dashboard. we will need data from many `applications` i'm caching the data here as well because i noticed when testing lots of executions i was looking at seem to call the same pipelines and fail in other applications etc

if you have better ideas i would love them, again i've only used spinnaker for about 2-3 weeks now and i have 0 experience with angular which seems to be the entire data layer